plan/pipeline.go

```diff
@@ -108,6 +108,11 @@ func (g *pipelineGen) PlanNode(taski interface{}) (interface{}, error) {
 	}
 	status, err := twrapper.Task.Check(renderer)
 
+	// create empty Status structure, if it not created in .Check()
```

Comment:
if we use this pr, we probably need the same check in apply/pipeline.go. That is where I am panicking in #343